plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/internal/model/commands/CompoundModelCommand.java

```diff
@@ -117,7 +118,8 @@ public static Command compose(EditingDomain editingDomain, List<Command> command
 			}
 			return result;
 		}
-		return UnexecutableCommand.INSTANCE;
+		/* if list is empty, simply do nothing but still allow execution */
+		return IdentityCommand.INSTANCE;
```

Comment:
What motivated this change?  In EMF, the `CompoundCommand` is not executable if it contains no commands.  So this would contradict that standard behaviour.

Reply:
I did this initially for the use case when no nudge commands have to be created and still want an executable command overall. But I actually already added the check to the NudgeOnRemovalCommand later on to return an IdentityCommand in case there are no nudge commands. So you are right, this should be reverted again.